packages/core/src/command/patient-import/patient-import-cloud.ts

```diff
@@ -123,44 +129,50 @@ export class PatientImportHandlerCloud implements PatientImportHandler {
         log(`Dryrun is true, returning...`);
         return;
       }
-      const outcomes = await Promise.allSettled(
-        patients.map(async patient => {
-          const patientPayload = createPatientPayload(patient);
-          const processPatientCreateRequest: ProcessPatientCreateEvemtPayload = {
-            cxId,
-            facilityId,
-            jobId,
-            jobStartedAt,
-            patientPayload,
-            rerunPdOnNewDemographics,
-          };
-          try {
-            await sqsClient.sendMessageToQueue(
-              processPatientCreateQueue,
-              JSON.stringify(processPatientCreateRequest),
-              {
-                fifo: true,
-                messageDeduplicationId: uuidv7(),
-                messageGroupId: cxId,
-              }
-            );
-          } catch (error) {
-            const msg = `Failure while sending payload to patient create queue @ PatientImport`;
-            log(`${msg}. Cause: ${errorToString(error)}`);
-            capture.error(msg, {
-              extra: {
-                cxId,
-                jobId,
-                processPatientCreateRequest,
-                context: "patient-import-cloud.send-payload-to-patient-create-queue",
-                error,
-              },
-            });
-            throw error;
-          }
-        })
-      );
-      const hadFailure = outcomes.some(outcome => outcome.status === "rejected");
+      const allOutcomes: PromiseSettledResult<void>[] = [];
+      const patientChunks = chunk(patients, patientCreateChunk);
+      for (const patientChunk of patientChunks) {
+        const chunkOutcomes = await Promise.allSettled(
+          patientChunk.map(async patient => {
+            const patientPayload = createPatientPayload(patient);
+            const processPatientCreateRequest: ProcessPatientCreateEvemtPayload = {
+              cxId,
+              facilityId,
+              jobId,
+              jobStartedAt,
+              patientPayload,
+              rerunPdOnNewDemographics,
+            };
+            try {
+              await sqsClient.sendMessageToQueue(
+                processPatientCreateQueue,
+                JSON.stringify(processPatientCreateRequest),
+                {
+                  fifo: true,
+                  messageDeduplicationId: uuidv7(),
+                  messageGroupId: cxId,
+                }
+              );
+            } catch (error) {
+              const msg = `Failure while sending payload to patient create queue @ PatientImport`;
+              log(`${msg}. Cause: ${errorToString(error)}`);
```

Comment:
hypernit: why create a const for a msg that's used once?

Reply:
you mean vs. one-line or vs. a "var" or something?